mantle/cmd/kola/options.go

```diff
@@ -252,7 +252,7 @@ func syncCosaOptions() error {
 		}
 	}
 
-	if kola.Options.IgnitionVersion == "" {
+	if kola.Options.IgnitionVersion == "" && kola.QEMUOptions.DiskImage == "" {
```

Comment:
It took me a while to debug why:
```
cosa run -d /path/to/rhcos-4.2.0.qcow2
```
was still getting spec3, it was because I was running from my `/srv/walters/fcos` builds directory...

I think in the future what we should do is basically synthesize a `Build` object if provided a `--qemu-image` and internally always reference that.